src/BrazeKit-dev.js

```diff
@@ -792,10 +794,13 @@ var constructor = function () {
         braze.addSdkMetadata(['mp']);
         primeBrazeWebPush();
 
-        if (forwarderSettings.userIdentificationType === 'MPID' && mParticle.Identity != null && mParticle.Identity.getCurrentUser().getMPID() != null) {
-            onUserIdentified(mParticle.Identity.getCurrentUser())
+        if (
+            mParticle.Identity != null &&
+            mParticle.Identity.getCurrentUser().getMPID() != null
+        ) {
+            onUserIdentified(mParticle.Identity.getCurrentUser());
```

Comment:
If there's a chance that `mParticle.Identity` can be null, then there's a chance that also `getCurrentUser()` can also return `null`.


```suggestion
        const currentUser = mParticle.Identity != null ? mParticle.Identity.getCurrentUser() : null;
        const mpid = currentUser.getMPID();
        
        if (currentUser && mpid) {
            onUserIdentified(currentUser);
```

Reply:
Looks like we are calling currentUser.getMPID() in your suggestion despite there being a chance it's null, so i'll use your code but update it a bit